src/game.h

```diff
@@ -1154,6 +1154,9 @@ class game
         // show NPC pathfinding on overmap ui
         bool debug_pathfinding = false; // NOLINT(cata-serialize)
 
+        //Ugly kludge to pass info to tile overmaps
+        npc *follower_path_to_show = nullptr;
```

Comment:
I really don't like this but the only alternative I see is passing display_path into a long chain of functions to get it to show up for tile overmaps.